packages/cli/src/ui/components/SettingsDialog.test.tsx

```diff
@@ -152,8 +151,32 @@ vi.mock('../../utils/settingsUtils.js', async () => {
 // const originalConsoleError = console.error;
 
 describe('SettingsDialog', () => {
+  // Simple delay function for remaining tests that need gradual migration
```

Comment:
🙏  do you want to open an issue to make sure we migrate the rest of the tests? (if you haven't already - or maybe you've already started on it 😁 )